addons/website/static/src/snippets/s_process_steps/000.scss

```diff
@@ -18,33 +60,90 @@
             .s_process_step_icon {
                 position: relative;
                 z-index: 1;
-                span:after {
-                    content: '';
-                    z-index: -1;
-                    border-top: 1px solid gray('500');
-                    @include o-position-absolute(50%, 0, 0, auto);
+                span.s_process_step_arrow {
+                    @include base_arrow;
+                    border-width: 2px;
                 }
-            }
-            .s_process_step_icon {
-                span:after {
-                    width: 100%;
+                span.s_process_step_curved_arrow {
+                    @include base_arrow;
+                    border-radius: 50%/40% 50% 0 0;
+                    border-width: 4px;
+                    top: -15%;
+                    &:after {
+                        top: auto;
+                        right: -0.75%;
+                        border-width: 20px 7px 0px 7px;
+                        transform: rotate(-60deg);
+                    }
                 }
             }
-            &:first-child .s_process_step_icon,
-            &:last-child .s_process_step_icon {
-                span:after {
-                    width: 50%;
+            &:nth-child(even) .s_process_step_icon {
+                span.s_process_step_curved_arrow {
+                    top: 70%;
+                    transform: rotate(180deg);
+                    &:after {
+                        left: -1.8%;
+                        top: -5%;
+                        transform: rotate(62deg);
+                    }
                 }
             }
             &:first-child .s_process_step_icon {
-                .fa:after { right: 0; }
-                .fa.float-right:after { width: 0; }
+                span.s_process_step_line_left {
+                    width: 0;
+                }
             }
             &:last-child .s_process_step_icon {
-                span:after { left: 0; }
-                .fa {
-                    &:after { left: 0; }
-                    &.float-left:after { width: 0; }
+                span.s_process_step_line_right {
+                    width: 0 !important;
+                    &:after {
+                        border-width: 0 !important;
+                    }
+                }
+            }
+            &.col-lg-6 {
+                @include override_curved_arrow($borderWidth: 6px);
+            }
+            &.col-lg-2,
+            &.col-lg-3 {
```

Comment:
Seems extremely strange to have rules that target *specific sizes* of columns. That should not be necessary. Also I tested the branch 1min and it's really not good enough:

![Screenshot from 2021-05-27 14-28-07](https://user-images.githubusercontent.com/10338094/119825931-c9657080-bef7-11eb-9f83-8a87b7a838e0.png)

-> The first arrow is not resized properly
-> The second arrow is resized but appear broken

Please test a lot more :wink: The result must not be "ok" in "some cases" but "perfect" in "all cases" :wink: